lib/checking_account.rb

```diff
@@ -0,0 +1,50 @@
+module Bank
+
+  class CheckingAccount < Account
+
+    def initialize(id, initial_balance, open_date)
+      super
+      @fee = 1.0
+      @check_number = 1
+    end
+
+    # CheckingAccount uses the withdraw method from Account
+    # This method does not need to be called here, but is left in for clarity
+    def withdraw(amount_to_withdraw)
+      super
```

Comment:
If you are going to call super you should just be able to leave this method definition off so that it will directly call the super method